vscode-lean4/test/suite/utils/helpers.ts

```diff
@@ -172,10 +172,9 @@ export async function restartLeanServer(client: LeanClient, retries=10, delay=10
     return false;
 }
 
-export async function assertLeanVersion(infoView: InfoProvider, version: string) : Promise<string> {
```

Comment:
This is more like "assertStringInInfoview" or something, right? Since e.g. `Lake Version:` is not itself a version, just a string that we look for.

Reply:
fixed